deps/build.jl

```diff
@@ -70,4 +70,8 @@ provides(BuildProcess, (@build_steps begin
     end
 end), [libfftw, libfftwf])
 
-BinDeps.@install Dict([:libfftw => :libfftw, :libfftwf => :libfftwf])
+if is_windows()
+    BinDeps.@install Dict([:libfftw => :libfftw3, :libfftwf => :libfftwf3])
+else
+    BinDeps.@install Dict([:libfftw => :libfftw3_threads, :libfftwf => :libfftwf3_threads])
```

Comment:
Then I think this should be `:libfftw => :libfftw` as it was before. That also doesn't get the deps.jl file to generate correctly though.